packages/rest-crud/README.md

```diff
@@ -64,9 +64,9 @@ class defined without the need for a repository or controller class file.
 
 If you would like more flexibility, e.g. if you would only like to define a
 default `CrudRest` controller or repository, you can use the two helper methods
-(`defineCrudRestController` and `defineCrudRepositoryClass`) exposed from
-`@loopback/rest-crud`. These functions will help you create controllers and
-respositories using code.
+(`defineCrudRestController` from `@loopback/rest-crud` and
+`defineCrudRepositoryClass` from `@loopback/repository`). These functions will
+help you create controllers and repositories using code.
```

Comment:
Should we revert this change since `defineCrudRepositoryClass` is still exported by `@loopback/rest-crud`? (Same comment applies to other similar doc changes below.)

Reply:
I'm fine either way.